Wrappers/Python/cil/optimisation/functions/IndicatorBox.py

```diff
@@ -32,50 +34,73 @@ class IndicatorBox(Function):
     
     '''
     
-    def __init__(self,lower=-numpy.inf,upper=numpy.inf):
+    def __init__(self,lower=-np.inf,upper=np.inf):
         '''creator
 
-        :param lower: lower bound
-        :type lower: float, default = :code:`-numpy.inf`
-        :param upper: upper bound
-        :type upper: float, optional, default = :code:`numpy.inf`
+        Parameters:
+        -----------
+        lower : float, DataContainer or numpy array, default ``-np.inf``
```

Comment:
could our default be `None`, that amounts to the same thing but is maybe a bit easier for the user if they only want to set one bound.